src/main/java/com/mapzen/valhalla/Instruction.kt

```diff
@@ -1,60 +1,59 @@
 package com.mapzen.valhalla
 
-import android.location.Location
-import android.util.Log
 import com.mapzen.helpers.DistanceFormatter
+import com.mapzen.model.Location
 import org.json.JSONException
 import org.json.JSONObject
 import java.util.Locale
 
-public open class Instruction {
+open class Instruction {
     companion object {
         val KM_TO_METERS = 1000
         val MI_TO_METERS = 1609.344
     }
 
-    public val NONE : Int = 0
-    public val START : Int = 1
-    public val START_RIGHT : Int = 2
-    public val START_LEFT : Int = 3
-    public val DESTINATION : Int = 4
-    public val DESTINATION_RIGHT : Int = 5
-    public val DESTINATION_LEFT : Int = 6
-    public val BECOMES : Int = 7
-    public val CONTINUE : Int = 8
-    public val SLIGHT_RIGHT : Int = 9
-    public val RIGHT : Int = 10
-    public val SHARP_RIGHT : Int = 11
-    public val U_TURN_RIGHT: Int = 12
-    public val U_TURN_LEFT : Int = 13
-    public val SHARP_LEFT : Int = 14
-    public val LEFT : Int = 15
-    public val SLIGHT_LEFT : Int = 16
-    public val RAMP_STRAIGHT : Int = 17
-    public val RAMP_RIGHT : Int = 18
-    public val RAMP_LEFT : Int = 19
-    public val EXIT_RIGHT : Int = 20
-    public val EXIT_LEFT : Int = 21
-    public val STAY_STRAIGHT : Int = 22
-    public val STAY_RIGHT : Int = 23
-    public val STAY_LEFT : Int = 24
-    public val MERGE : Int = 25
-    public val ROUNDABOUT_ENTER : Int = 26
-    public val ROUNDABOUT_EXIT: Int = 27
-    public val FERRY_ENTER: Int = 28
-    public val FERRY_EXIT: Int = 29
+    val NONE : Int = 0
```

Comment:
It seems these constant values for maneuver types are no longer used in the library or the app and _could_ be safely removed.

Although I wonder if its better to keep them in as documentation of the possible return values for `getIntegerInstruction()`?

In which case we should at least move these into the `companion object` so they are static class members rather than fields on each instance of `Instruction`.